.circleci/config.yml

```diff
@@ -201,12 +201,4 @@ workflows:
           filters:
             branches:
               only: master
-      # Ensure that build_docs_pdf always runs last, so that we can use the CircleCI API link for the "latest" artifact
-      # https://circleci.com/api/v1.1/project/github/bids-standard/bids-specification/latest/artifacts/0/bids-spec.pdf?branch=master
-      - build_docs_pdf:
-          requires:
-            - build_docs
-            - linkchecker
-            - github-changelog-generator
-            - remark
-            - Changelog-bot
```

Comment:
this is no longer needed (and hasn't been needed for a long time actually), because we use https://github.com/larsoner/circleci-artifacts-redirector-action

It's good we finally change this, because now we can see whether PDF build passes when linkchecker doesn't ... and linkchecker is arguably the test that fails most often, which often prevented us from seeing the PDF check